modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -78,13 +88,312 @@ class NDB_Form_statistics extends NDB_Form
         //$DB =& Database::singleton();
     }
 
+    function stats_demographic()
+    {
+        $DB =& Database::singleton();
+
+        //if table is not in database handle by displaying error message
+        $this->tpl_data['mri_table_exists'] = true;
+        if (!$DB->tableExists('mri_parameter_form')) {
+            $this->tpl_data['mri_table_exists'] = false;
+            return;
+        }
+
+        $this->tpl_data['showTable'] = true;
+
+        //PROJECTS
+        $projects[null] = 'All Projects';
+        foreach(Utility::getProjectList() as $key => $value) {
+            $projects[$key] = $value;
+        }
+        $currentProject = null;
+        if(isset($_REQUEST['DemographicProject']) && $_REQUEST['DemographicProject'] != '') {
+            $ExtraProject_Criteria = ' AND (c.ProjectID IS NULL OR c.ProjectID=' .
+                $_REQUEST['DemographicProject'] . ')';
```

Comment:
This is directly taking user input and concatenating it into an SQL statement. It's very insecure and should use prepared statements instead.